### ISSUE TITLE
Split (.666xx or .0) PS2 ISO games on FAT32 MS/SD/CF are not detected when updating XML

### ISSUE THREAD

**Issue by r3v0lv3rz** [aldostools/webMAN-MOD#343]
As title.

I can see them when using ManaGunZ or multiMAN and can then manually copy and join them together when copying to the internal drive but webMAN-MOD does not.

As I understand it PS3 ISOs do not suffer from this limitation, can the code be adapted to include PS2ISO as well?

**Comment by aldostools** [comment]
PS2 ISOs only works from hdd0.

Multipart ISO games should be named *.iso.0 to *.iso.15 depending on the
number of parts.

webMAN offers a very limited support for .666xx files.

**Comment by r3v0lv3rz** [comment]
> PS2 ISOs only works from hdd0.

Yes, am aware and when using FAT32 on USB/MS/CF/SD with PS2ISO <4GB there is no issue and webMAN-MOD will detect the ISO fine when updating the XML. I can select it from XMB and then the ISO is copied to hdd0 for use.

> Multipart ISO games should be named *.iso.0 to *.iso.15 depending on the number of parts.

OK, I tried this with a PS2ISO >4GB, which is on /dev_ms/PS2ISO/ and still it is not detected by webMAN-MOD when refreshing the XML.

**Comment by Joonie86** [comment]
No further feedback
